src/components/molecules/actionables.tsx

```diff
@@ -28,7 +28,7 @@ const Actionables: React.FC<ActionablesProps> = ({ actions }) => {
     <div>
       <DropdownMenu.Root>
         <DropdownMenu.Trigger asChild>
-          <Button variant="ghost" size="medium">
+          <Button variant="ghost" size="small" className="py-0">
             <MoreHorizontalIcon />
```

Comment:
According to the design this should be a square icon 32 x 32 with horizontal dots of size 20. I suggest we do something like this, but if you fancy we can go with fixed widths as well ie. `w-[32px]`
```suggestion
          <Button 
            variant="ghost" 
            size="small" 
            className="w-8 h-8 p-1.5 focus:border-none focus:shadow-none"
          >
            <MoreHorizontalIcon size={20} />
```
```

Reply:
definitely not inline with the design, but the issue is that the design of the button (if it is set to 32x32) makes it so large that it pushes everything else out of place. I could do something like setting the positioning to absolute so it does not affect everything around it, but in that case we would have to do some weird magic numbers to make sure that it sits in the proper position. The issue can be seen on this image, as this does not translate that well into css:

![image](https://user-images.githubusercontent.com/45367945/150503573-4cfdc375-491e-4bc9-862d-f3102db11204.png)

But changing the entire look of Actionables was definitely not a good idea 😅 I've made a change that brings it closer in line with the design without changing the look of Actionables, and then I will check in with Ludvig, but I think this should work just fine for now 😄